usaspending_api/download/v2/request_validations.py

```diff
@@ -143,15 +194,19 @@ def _validate_award_id(request_data):
     if award_id_type not in (str, int):
         raise InvalidParameterException("Award id must be either a string or an integer")
     if award_id_type is int or award_id.isdigit():
-        award = Award.objects.filter(id=int(award_id), type__startswith="IDV").values_list("id", "piid").first()
```

Comment:
it's possible to move the filter args to a dictionary and define the queryset once:
```
if award_id_type is int or award_id.isdigit():
    filters = {"id: int(award_id)}
else:
    filters = {"generated_unique_award_id": award_id}

award = Award.objects.filter(**filters).values_list("id", "piid", "fain", "uri", "generated_unique_award_id").first()
```